### PR TITLE
add first details about OpenShift

### PR DESCRIPTION
**Pull request by pgerber**
• OpenShift/Docker basics
• describe how a setup looks like for a customer
• introduction to oc
• add glossary

### CODE REVIEW

File: openshift/basics.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+Introduction to OpenShift
+-------------------------
+
+`OpenShift`_ provides a management tool to deploy `Docker`_ containers.
+
+Here's what's important:
+
+* A single Docker Image image is deployed when continuous delivering. It contains Nice2, Java and all runtime
```
Comment (tstjep) [inline openshift/basics.rst]: -delivering
+delivery

File: openshift/basics.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  dependencies. Thus, Nice2 can be run in any OpenShift/Docker environment.
+* OpenShift orchestrates the Docker images, service, routes, etc. and ensure they stay available.
```
Comment (tstjep) [inline openshift/basics.rst]: ensures

File: openshift/basics.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
++------------------------------+-----------------------------------+----------------------------------------+---------------------------------+
+
+
+.. rubric:: Footnotes
+
+.. [#f1] Production systems use the ``production`` tag and test systems the ``test`` tag.
+.. [#f2] By default the ``stable`` tag is used. ``latest`` is the staging area and is only deployed on selected systems.
+.. [#f3] Mounted at ``/persist`` and only the subdirectory ``/persist/index_data`` is currently used for the Solr index.
+.. [#f4] Image source is hosted on `github <https://github.com/tocco/openshift-nginx>`_ and the ``latest`` tag is
+         automatically build on `dockerhub <https://hub.docker.com/r/toccoag/openshift-nginx/>`_.
```
Comment (tstjep) [inline openshift/basics.rst]: built?

File: openshift/basics.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+.. [#f5] Image source is hosted on `github <https://github.com/tocco/openshift-solr>`_ and the ``latest`` tag is
+         automatically build on `dockerhub <https://hub.docker.com/r/toccoag/openshift-solr/>`_.
```
Comment (tstjep) [inline openshift/basics.rst]: built?

File: openshift/edit_resources.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+NICE2_APP_*          Add custom entries to ``application.local.properties``.
+NICE2_HIKARI_*       Add custom entries to ``hikiricp.local.properties``.
+NICE2_JAVA_PARAM_*   Pass custom parameters to Java.
+NICE2_NICE_ARG_*     Pass custom argument to Nice.
+FLUENTD_TARGET_HOST  Target host for fluentd. (Fluentd forwarding is enabled automatically if this variable is set)
+FLUENTD_TARGET_PORT  Target port for fluentd. (defaults to 24224)
+===================  ===================================================================================================
+
+.. important::
+    Openshift does not currently allow '.' or '-' to appear as key of an environment variable. [#f2]_
```
Comment (tstjep) [inline openshift/edit_resources.rst]: keys?

File: openshift/edit_resources.rst [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+    As workaround:
+
+        ==============  ===========================
+        instead of      use
+        ==============  ===========================
+        ``.`` (period)  ``__`` (double underscore)
+        ``-`` (hypen)   ``___`` (triple underscore)
```
Comment (tstjep) [inline openshift/edit_resources.rst]: hyphen

File: openshift/basics.rst (hunk 1 above)
Comment (pgerber) [inline openshift/basics.rst]: don't think "delivery" works here, kinda need a verb here. Could use "… when executing a continuous delivery". Better?

File: openshift/basics.rst (hunk 2 above)
Comment (pgerber) [inline openshift/basics.rst]: agreed

File: openshift/basics.rst (hunk 3 above)
Comment (pgerber) [inline openshift/basics.rst]: will change it

File: openshift/basics.rst (hunk 4 above)
Comment (pgerber) [inline openshift/basics.rst]: irregular verbs should be banned

File: openshift/edit_resources.rst (hunk 5 above)
Comment (pgerber) [inline openshift/edit_resources.rst]: Don't think it really matters. Do you? If "keys" it used should also be "variables" I guess.

File: openshift/edit_resources.rst (hunk 6 above)
Comment (pgerber) [inline openshift/edit_resources.rst]: indeed

File: openshift/basics.rst (hunk 1 above)
Comment (tstjep) [inline openshift/basics.rst]: you're right, maybe "when deploying with/through/via continuous delivery?"

File: openshift/edit_resources.rst (hunk 5 above)
Comment (tstjep) [inline openshift/edit_resources.rst]: you're probably right.

**Pull request merged**